README.md

```diff
@@ -39,31 +96,42 @@ You can pass additional arguments to Theia after the image name, for example to
 
     # Linux, macOS, or PowerShell
     docker run -it --init -p 3000:3000 --expose 9229 -p 9229:9229 -v "$(pwd):/home/project:cached" theiaide/theia:next --inspect=0.0.0.0:9229
-    
+
     # Windows (cmd.exe)
     docker run -it --init -p 3000:3000 --expose 9229 -p 9229:9229 -v "%cd%:/home/project:cached" theiaide/theia:next --inspect=0.0.0.0:9229
 
-### Image Variants
+### Electron Apps
 
-#### `theiaide/theia:latest`
+<img
+    src='./assets/electron.png'
+    alt='electron logo'
+    width="50px"
+/>
 
-This image is based on the latest stable released version.
+| Application Name | Description | Documentation |
+|:---|:---|:---|
+| `theia-cpp-electron` | Theia-based C/C++ desktop IDE | [docs](./theia-cpp-electron/README.md) |
+| `theia-electron` | Theia-based JavaScript/TypeScript (Web Technologies) desktop IDE | [docs](./theia-electron/README.md) |
 
-#### `theiaide/theia:<version>`
+---
 
-This image is based on the given stable released version.
+<br />
 
-#### `theiaide/theia:next`
+## Plugins
 
-This image is based on the nightly published version.
+Theia supports VS Code extensions as plugins. If a given image has plugin support, one can
```

Comment:
This is a bit vague. Since we recently added `@theia/vsx-registry` to most images, we could mention that instead. Something like : 
"Many of the applications in this repository now include support for running VS Code extensions and an integration to the public instance of [open-vsx registry](https://open-vsx.org/), an open alternative to the Visual Studio Marketplace". Look for the "Extensions" view under the "Views" top-level menu.